.github/workflows/build-commitfest.yml

```
@@ -15,7 +15,7 @@ on:
 # set up environment variables to be used across all the jobs
 env:
   REGISTRY: "ghcr.io/${{ github.repository_owner }}/postgresql-trunk"
-  TAG: "commitfest-master"
+  TAG: "${{ vars.PG_MAJOR }}-commitfest-master"
```

```suggestion
  TAG: "${{ vars.PG_MAJOR }}-commitfest-master"
```

Why master here @NiccoloFei ?